src/index.rst

```diff
@@ -127,38 +136,21 @@ The project maintains two email lists:
   * 'axom-dev@llnl.gov' is for communication among team members. 
 
 
--------------- 
+---------
 Chat Room
--------------- 
+---------
 
 We also have a chat room on LLNL's Cisco Jabber instance called 
```

Comment:
Should we remove the jabber chat room part and only mention MS Teams?

Reply:
Yes, banish jabber chat to the collaborative software scrapheap!